app/db.py

```diff
@@ -48,13 +49,16 @@ async def _is_initialized(self, db: aiosqlite.Connection) -> bool:
             return False
 
     async def _run_init_scripts(self, db: aiosqlite.Connection):
-        """Execute all scripts frob db/init"""
-        init_dir=Path(__file__).parent.parent / "db" / "init"
-
+        """Execute all scripts from db/init"""
+        init_dir = Path(os.getenv("DB_INIT_DIR", "/data/init"))
+        if not init_dir.exists():
+            logging.error(f"Init directory does not exist: {init_dir}")
+            return
```

Comment:
It is uneccessary. You use enviroment variable that you are not even setting anywhere.

```suggestion
        init_dir = "/data/init"
        if not init_dir.exists():
            logging.error(f"Init directory does not exist: {init_dir}")
            return
```